documentation/en/api-experimental-user-facing-v2-docs/api-v2-experimental.md

```diff
@@ -441,6 +430,41 @@ The method returns the ID address (string) corresponding to the input address at
 
 If the address cannot be resolved to an ID address at the specified tipset state (e.g., it never existed or was created after that tipset), an error is returned.
 
+## Ethereum (Eth)
+
+The V2 API namespace includes all the Ethereum (ETH) APIs that `/v1` supports, but the `finalized` and `safe` tags have been updated to consult the F3 subsystem (if enabled) for finality information.  `finalized` and `safe` for the "Eth" APIs has the same meaning and fallback outlined in  [Consensus Protocol Notes](#consensus-protocol-notes).
```

Comment:
Worth nothing the different distance semantics for `safe` in Eth API v2? i.e. 200 distance instead of 30 in v1.

Reply:
Done in https://github.com/filecoin-project/lotus/pull/13068/commits/02c2783214210999d3e26baca0892028809dfac6 .  Resolving.